be/src/io/fs/file_writer.h

```diff
@@ -33,8 +33,8 @@ class FileSystem;
 struct FileWriterOptions {
     bool write_file_cache = false;
     bool is_cold_data = false;
-    bool sync_file_data = true;        // Whether flush data into storage system
-    int64_t file_cache_expiration = 0; // Absolute time
+    bool sync_file_data = true;         // Whether flush data into storage system
+    uint64_t file_cache_expiration = 0; // Absolute time
```

Comment:
why do we need to change the type?

Reply:
Because the table property `file_cache_ttl_seconds` is long in FE, it maybe set the LONG_MAX value. file_cache_expiration = cur_time + file_cache_ttl_seconds, it will overflow if not uint64_t

Reply:
just keep it LONG_MAX when detect overflow